src/application-manager/types.ts

```diff
@@ -3,34 +3,34 @@ import { BrowserWindow } from 'electron'
 import { FlowrWindow } from '../frontend/flowr-window'
 import { Store } from '../frontend/src/store'
 
-export interface ApplicationInitConfig {
+interface ApplicationInitConfig {
   application: FlowrApplication
   capabilities?: {[key: string]: boolean}
   config?: ApplicationConfig
 }
 
-export interface ApplicationCanOpenConfig {
+interface ApplicationCanOpenConfig {
   application: FlowrApplication
   config?: {[key: string]: any}
 }
 
-export interface ApplicationOpenConfig {
+interface ApplicationOpenConfig {
```

Comment:
Why do we have two identical interface: ApplicationCanOpenConfig and ApplicationOpenConfig ?